visual-java/src/main/java/com/saucelabs/visual/CheckOptions.java

```diff
@@ -31,9 +32,11 @@ public CheckOptions(
       Boolean captureDom,
       String clipSelector,
       WebElement clipElement,
-      FullPageScreenshotConfig fullPageScreenshotConfig) {
+      FullPageScreenshotConfig fullPageScreenshotConfig,
+      List<IgnoreSelectorIn> ignoreSelectors) {
```

Comment:
We need an extra constructor that does not require `ignoreSelector` parameter.

Since this is was public one, there is no way to ensure that no use is using it. And in that case, it will introduce a breaking change.